components/__tests__/__snapshots__/Currency.test.js.snap

```diff
@@ -33,6 +33,26 @@ exports[`Currency currency format with separators 3`] = `
 </span>
 `;
 
+exports[`Currency overrides precision if amount is below 100 1`] = `
+<span
+  className="Text__P-sc-3suny7-0-span hdWHPl"
+  fontSize="inherit"
+  letterSpacing="-0.4px"
+>
+  ¥0.99
```

Comment:
This is not correct, if it's a zero decimal currency then the expected display is:
```suggestion
  ¥99
```